app/models/work.rb

```diff
@@ -101,6 +101,7 @@ def dates
   def self.get_query_url(options={})
     if options[:id].present?
       params = { q: options[:id],
+                 fq: "doi:#{options[:id].dump}",
```

Comment:
What is the `dump` providing?

Reply:
It escapes characters. So the DOI name is escaped and then URL encoded ( a few lines down).

Reply:
What happens when you remove `dump`? As we are not doing this in many similar places in our code.

Reply:
@kjgarza why is `dump` needed here?